jsapp/js/components/permissions/transferProjects/transferModalWithBanner.tsx

```diff
@@ -0,0 +1,74 @@
+// Libraries
+import React, {useState, useEffect} from 'react';
+import {useSearchParams} from 'react-router-dom';
+
+// Partial components
+import TransferProjectsInvite from './transferProjectsInvite.component';
+import InviteBanner from './inviteBanner';
+
+// Stores, hooks and utilities
+import {
+  isInviteForLoggedInUser,
+  type TransferStatuses,
+} from './transferProjects.api';
+
+// Constants and types
+import type {InviteState} from './inviteBanner';
+
+/**
+ * This is a glue component that displays a modal from `TransferProjectsInvite`
+ * and a banner from `InviteBanner` as an outcome of the modal action.
+ */
+export default function TransferModalWithBanner() {
```

Comment:
Similar to my comment about invitations above, we are going to have organization ownership transfers down the line. Should we rename this something like ProjectOwnershipTransferModalWithBanner?